aviary/interface/methods_for_level2.py

```diff
@@ -836,36 +840,38 @@ def _get_phase(self, phase_name, phase_idx):
                 user_options.set_val('initial_ref', init_bounds[0][1],
                                      units=init_bounds[1])
             else:
-                user_options.set_val('initial_ref', 10., 'min')
+                user_options.set_val('initial_ref', 600., time_units)
```

Comment:
Should we do something to make sure that the ref value is something reasonable?
```
time_val = openmdao.utils.units.convert_units(600, 's', new_units=time_units)
user_options.set_val('initial_ref', time_val, time_units)
```

Reply:
I really don't know the provenance of this line, or why it is needed. We can't use an actual unit here, because some of our phases have a custom integration variable with different units, and we will get an exception if the ref has units incompatible with the other "time" units.